### PR TITLE
Add Konnek event source and SinkBinding

### PR DESCRIPTION
**Pull request by jonatasbaldin**
Fixes #issue-number or description of the problem the PR solves

## Proposed Changes 
- Adding a new section into Eventing > Source Samples for `SinkBindings`. I think this is a good idea, but would love any feedback :)
- Add Konnek in the `SinkBinding` section

### CODE REVIEW

**Comment by jonatasbaldin** [comment]
The failed tests looks like some Trigger Mesh links... gonna flag @sebgoa here just to let him know :)

**Comment by lionelvillard** [comment]
Pinging @antoineco as well.

**Comment by antoineco** [comment]
Oops, nice catch! Those sources are now located at https://github.com/triggermesh/aws-event-sources

**Comment by sebgoa** [comment]
fix got merged: https://github.com/knative/docs/commit/65b7cd8b6be64080263432c2fdcc5dcdc971fe38

File: docs/eventing/sources/generator/ReadmeTemplate.gomd [hunk 1]
```diff
@@ -64,3 +64,13 @@ Name | Status | Support | Description
 {{ range .Containers -}}
 [{{ .Name }}]({{ .Url }}) | {{ .Status }} | {{ .Support }} | {{ .Description }}
 {{- end }}
+
+### SinkBindings
+
+These are containers intended to be used with `SinkBinding`.
```
Comment (evankanderson) [inline docs/eventing/sources/generator/ReadmeTemplate.gomd:70]: Link to the SinkBinding documentation.

File: docs/eventing/sources/sources.yaml [hunk 2]
```diff
@@ -202,3 +202,12 @@ containers:
...
       Watches for files being uploaded into a FTP/SFTP and generates events for those.
+
+# These are intended to be used with SinkBinding, but are not CRDs.
+sinkBindings:
+  - name: Konnek
+    url: https://konnek.github.io/docs/#/
+    status: Active Development
+    support: None
+    description: >
+      Retrieves events from cloud platforms – like AWS and GCP –, transform them into CloudEvents and consume in Knative using SinkBinding.
```
Comment (evankanderson) [inline docs/eventing/sources/sources.yaml:213]:
```suggestion
      Retrieves events from cloud platforms (like AWS and GCP) and transforms them into CloudEvents for consumption in Knative.
```

File: docs/eventing/sources/generator/ReadmeTemplate.gomd (hunk 1 above)
Comment (jonatasbaldin) [reply docs/eventing/sources/generator/ReadmeTemplate.gomd]: Added!

**Pull request merged**